packages/style-engine/class-wp-style-engine.php

```diff
@@ -594,40 +578,21 @@ public function compile_css( $css_declarations, $css_selector ) {
 		if ( $css_selector ) {
 			$css_rule = new WP_Style_Engine_CSS_Rule( $css_selector, $css_declarations );
 			return $css_rule->get_css();
-		} else {
-			$css_declarations = new WP_Style_Engine_CSS_Declarations( $css_declarations );
-			return $css_declarations->get_declarations_string();
-		}
-	}
-
-	/**
-	 * Returns a string of classnames,
-	 *
-	 * @param string $classnames A flat array of classnames.
-	 *
-	 * @return string A string of classnames separate by a space.
-	 */
-	public function compile_classnames( $classnames ) {
```

Comment:
The `compile_classnames` method was only used in 1 spot, and it just does an `implode( ' ', array_unique( $classnames ) );` so nothing complicated that would warrant a new method. The implode & array_unique were simply added to the place where this method was being called.

Reply:
Good call